src/main/scala-2.11/markets/orders/limit/LimitBidOrder.scala

```diff
@@ -30,6 +31,11 @@ case class LimitBidOrder(issuer: ActorRef,
                          tradable: Tradable,
                          uuid: UUID) extends LimitOrderLike with BidOrder {
 
+  def crosses(order: AskOrder): Boolean = order match {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Default case not specified.](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1843449467/issues/source?bid=2921361&fileBranchId=3082852#l34)